### DISCUSSION TITLE
Replace move maps on last update.

### DISCUSSION THREAD

**Discussion by Thomas-Bagnolati** [JetBrains/ideavim#423] [Q&A] [answered]
Hi, I'm using ideavim on android studio.

With the last update something when wrong with my previous keymap, maybe someone can help me to resolve it ?

I want to move fast with:
- <C-h> like <b> default vim key.
- <C-l> like <w> default vim key.

It's was woking perfectly before the last update of the plugin, this is the part on my ideavimrc file:

```vim
" Go to next/previous word
nnoremap <C-h> <b>
nnoremap <C-l> <w>
vnoremap <C-h> <b>
vnoremap <C-l> <w>
nnoremap <C-S-h> <B>
nnoremap <C-S-l> <W>
vnoremap <C-S-h> <B>
vnoremap <C-S-l> <W>
```

Now when I try to use this remaps, it make some indent tab. I don't really understand what is goin on.

Thanks for reading, Thomas.

**Comment by lippfi** [comment, marked as answer]
Hi, Thomas!
You shouldn't place keystrokes in triangle brackets, please replace your config with following lines:
```
" Go to next/previous word
nnoremap <C-h> b
nnoremap <C-l> w
vnoremap <C-h> b
vnoremap <C-l> w
nnoremap <C-S-h> B
nnoremap <C-S-l> W
vnoremap <C-S-h> B
vnoremap <C-S-l> W
```
For Vim `<b`/`<w`  is  `<{motion}` and does a completely different thing than going to the previous/next word  (see `:h <`)
